doc/console/SONiC-Console-Switch-High-Level-Design.md

```diff
@@ -203,16 +242,19 @@ For A, user need to specific the line number by following a colon after username
 ssh tom:1@host
 ```
 
-By default, the content between ```:``` and ```@``` will be trimed by sshd before it do authentication and the trimed string will be dropped silently. To use this segment as line number for reverse SSH feature, we need to modify the source ode of OpenSSH and put this segment to a environment variable ```$SSH_TARGET_CONSOLE_LINE```, then we can insert a short script into ```/etc/bash.bashrc``` and run command ```consutil connect $SSH_TARGET_CONSOLE_LINE``` to enter the management session automatically after user login.
+By default, the content between `:` and `@` will be trimmed by sshd before it do authentication and the trimmed string will be dropped silently. To use this segment as line number for reverse SSH feature, we need to modify the source ode of OpenSSH and put this segment to a environment variable `$SSH_TARGET_CONSOLE_LINE`, then we can insert a short script into `/etc/bash.bashrc` and run command `consutil connect $SSH_TARGET_CONSOLE_LINE` to enter the management session automatically after user login.
```

Comment:
minor issue, environment variable should be SSH_TARGET_CONSOLE_LINE (without $). Later reference can come with $